File: MPI/mpicheck/tests/functions/2.0/MPI_Comm_spawn_multiple.f90

        
        program main
        use mpi
        
        INTEGER var_0
       CHARACTER*(10) var_1
       CHARACTER*(10) var_2
       INTEGER var_3(10)
       INTEGER var_4(10)
       INTEGER var_5
       INTEGER var_6
       INTEGER var_7
       INTEGER var_8
       INTEGER var_9
        call mpi_comm_spawn_multiple(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, &
       var_8, var_9)
       call pmpi_comm_spawn_multiple(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, &
       var_8, var_9)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Comm_spawn_multiple.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    char *var_1[2];
    char **var_2[2];
    const int var_3[2];
    const MPI_Info var_4[2];
    int var_5;
    MPI_Comm var_6;
    MPI_Comm *var_7;
    int var_8[2];
    int ret;
    /* calls */
    ret = MPI_Comm_spawn_multiple(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    ret = PMPI_Comm_spawn_multiple(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    return 0;
}
